R/escape.R

```diff
@@ -229,13 +228,37 @@ sql_escape_logical <- function(con, x) {
   UseMethod("sql_escape_logical")
 }
 
+#' @keywords internal
+#' @export
+sql_escape_Date <- function(con, x) {
+  UseMethod("sql_escape_Date")
+}
+
+
+#' @keywords internal
+#' @export
+sql_escape_POSIXt <- function(con, x) {
+  UseMethod("sql_escape_POSIXt")
+}
+
 # DBIConnection methods --------------------------------------------------------
 
 #' @export
 sql_escape_string.DBIConnection <- function(con, x) {
   dbQuoteString(con, x)
 }
 
+#' @export
+sql_escape_Date.DBIConnection <- function(con, x) {
+  dbQuoteString(con, x)
+}
+
+#' @export
+sql_escape_POSIXt.DBIConnection <- function(con, x) {
+  x <- strftime(x, "%Y-%m-%dT%H:%M:%OSZ", tz = "UTC")
+  dbQuoteString(con, x)
```

Comment:
Should this be `%OS6`? DBI needs to be fixed here -- I think passing the time zone might be correct here.

```suggestion
  dbQuoteLiteral(con, x)
```

Reply:
This specific formatting is what is in place in dbplyr at the moment for POSIXt, so I wouldn't want to touch that.